src/umpire/device_allocator_helper.hpp

```diff
@@ -0,0 +1,83 @@
+////////////////////////////////////////////////////////////////////////////
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC and Umpire
+// project contributors. See the COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (MIT)
+////////////////////////////////////////////////////////////////////////////
+#ifndef UMPIRE_device_allocator_helper_HPP
+#define UMPIRE_device_allocator_helper_HPP
+
+#include <string.h>
+
+#include "umpire/DeviceAllocator.hpp"
+
+namespace umpire {
+
+/*
+ * Const variable for the limit of unique DeviceAllocator
+ * objects available at once.
+ */
+extern __managed__ int UMPIRE_TOTAL_DEV_ALLOCS;
+
+/*
+ * Global arrays for both host and device which hold the
+ * DeviceAllocator objects created.
+ */
+extern DeviceAllocator* UMPIRE_DEV_ALLOCS_h;
+__device__ extern DeviceAllocator* UMPIRE_DEV_ALLOCS;
+
+/*
+ * Get the DeviceAllocator object specified by either the given
+ * name or id.
+ */
+__host__ __device__ DeviceAllocator getDeviceAllocator(const char* name);
```

Comment:
Can we make these snake_case?

Reply:
Done for the device_allocator_helper functions, the DeviceAllocator member functions stayed the same